main.tf

```diff
@@ -1,72 +1,111 @@
 resource "aws_route53_record" "a_records" {
-  count   = "${length(var.a_records)}"
-  name    = "${element(keys(var.a_records), count.index)}"
-  ttl     = "${var.ttl}"
+  count   = length(var.a_records)
+  name    = element(keys(var.a_records), count.index)
+  ttl     = var.ttl
   type    = "A"
-  zone_id = "${var.zone_id}"
+  zone_id = var.zone_id
 
-  records = ["${element( values( var.a_records ), count.index )}"]
+  # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
```

Comment:
Is this todo supposed to be in the PR?

Reply:
just tested the usage, did not expect the auto-update to inject todos
in our case the brackets are necessary though, since the type is map(string) and not map(list(string))
will remove them